packages/okta-react-native/index.js

```diff
@@ -0,0 +1,94 @@
+import { NativeModules, Platform } from 'react-native';
+import {
+	assertIssuer,
+  assertClientId,
+	assertRedirectUri
+} from '@okta/configuration-validation';
+import jwt from 'jwt-lite';
+
+export const createConfig = async (
+  clientId,
+  redirectUri, 
+  endSessionRedirectUri, 
+  discoveryUri,
+  scopes) => {
+
+  assertIssuer(discoveryUri);
+  assertClientId(clientId);
+  assertRedirectUri(redirectUri);
+  assertRedirectUri(endSessionRedirectUri);
+
+  if (Platform.OS === 'ios') {
+		scopes = scopes.join();
+		scopes = scopes.replace(/,/g, ' ');
```

Comment:
`.join()` can take an argument for the separator, which defaults to ','.  If you said
`scopes = scopes.join(' ');` above, this line would be unnecessary.